tests/SelfTest/IntrospectiveTests/String.tests.cpp

```diff
@@ -35,7 +35,17 @@ TEST_CASE( "StringRef", "[Strings][StringRef]" ) {
         REQUIRE_THROWS(original.c_str());
         REQUIRE_NOTHROW(original.data());
     }
-
+    SECTION( "Copy construction" ) {
```

Comment:
I would prefer "Copy construction is shallow" or "Copy constructor is shallow"

Reply:
My primary concern was if it by chance doesn't perform conversion to (a temporary) `std::string` and then construct a new `StringRef` from that (which it didn't, because the conversion operator is `explicit`). So maybe "Copy construction doesn't construct a temporary std::string"?

Reply:
Changed to "Copy construction is shallow".